client/post-editor/controller.js

```diff
@@ -235,6 +248,13 @@ module.exports = {
 		renderEditor( context, postType );
 	},
 
+	revision: function( context ) {
+		const siteId = getSelectedSiteId( context.store.getState() );
+		const postId = getPostID( context );
+
+		renderRevision( context, siteId, postId );
```

Comment:
Shouldn't these be `revisions` (current method) and `renderRevisions` (current line)?